packages/ref-metrics-ilamb/src/cmip_ref_metrics_ilamb/standard.py

```diff
@@ -111,10 +111,7 @@ def _load_csv_and_merge(output_directory: Path) -> pd.DataFrame:
     Load individual csv scalar data and merge into a dataframe.
     """
     df = pd.concat(
-        [
-            pd.read_csv(output_directory / f, keep_default_na=False, na_values=["NaN"])
-            for f in output_directory.glob("*.csv")
-        ]
+        [pd.read_csv(f, keep_default_na=False, na_values=["NaN"]) for f in output_directory.glob("*.csv")]
```

Comment:
@nocollier FYI: output_directory.glob returns a path that includes the output directory